lib/modules/manager/cargo/locked-version.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { DirectoryResult, dir } from 'tmp-promise';
+import { join } from 'upath';
+import { Fixtures } from '../../../../test/fixtures';
+import { fs } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import type { RepoGlobalConfig } from '../../../config/types';
+import { extractLockFileVersions, parseLockFile } from './locked-version';
+
+jest.mock('../../../util/fs');
+
+function mockReadLocalFile(files: Record<string, string | null>) {
+  fs.readLocalFile.mockImplementation((file): Promise<any> => {
+    let content: string | null = '';
+    if (file in files) {
+      content = files[file];
+    }
+    return Promise.resolve(content);
+  });
+}
+
+describe('modules/manager/cargo/locked-version', () => {
+  describe('extractLockFileVersions()', () => {
+    let adminConfig: RepoGlobalConfig;
+    let tmpDir: DirectoryResult;
+
+    beforeEach(async () => {
+      tmpDir = await dir({ unsafeCleanup: true });
```

Comment:
do you still need this?

Reply:
Removed in c3d2a226587df35820d9be89d5fefab2c88f0f44.